aiida_firecrest/transport.py

```diff
@@ -233,7 +233,8 @@ def _dynamic_info_direct_size(
 
 
 class FirecrestTransport(Transport):
-    """Transport interface for FirecREST."""
+    """Transport interface for FirecREST.
+    It must be used together with the 'firecrest' scheduler plugin."""
```

Comment:
Have you considered overwriting class variable `__doc__`? Then you can in principle also include this if you can retrieve auth_options is from another class

```python
class AuthoptionHolder:
    auth_options = {"machine": "the name of the machine"}

class A:
    __doc__ = ( """Transport interface for FirecREST.
    It must be used together with the 'firecrest' scheduler plugin."""
    +
    "Authentication parameters:\n"
        ) + "\n".join(
            [f"  {k}: {v}" for k, v in AuthoptionHolder.auth_options.items()]
        )

A.__doc__
```

Reply:
Thanks @agoscinski , it's interesting.
But it doesn't work, because `auth_options` should be from the current class `cls`, not the one we inherit from. 
So it easily turns into some fuss: one has to assign put the `__doc__` after `_valid_auth_options`, and do some list/tuple/dict magic to take the info out.  

Probably it's not that important. I don't thing if many people would like to see inputs of a class through `verdi plugin list` anyways.

Reply:
I think the description is not so important, but class documentation I think is important, since you get it with `help(<CLASS_NAME>)`.  But my solution also complexifies code logic so I am ok to not do it.